src/components/KMultiselect/KMultiselect.vue

```diff
@@ -323,22 +339,22 @@ const props = defineProps({
    */
   width: {
     type: String,
-    default: '',
+    default: '100%',
   },
   /**
    * Number of rows of selections to show when focused
    */
   selectedRowCount: {
     type: Number,
-    default: 2,
+    default: 1,
```

Comment:
question: why is the default changing?

Reply:
Because I figured from Figma designs that by default we only want 1 row of selections